src/Components/Politicians/PoliticianCard/index.js

```diff
@@ -23,10 +22,10 @@ const PoliticianCard = (props: Props) => {
           style={styles.flexItem}
         />
         <p style={styles.name}>{politician.name}</p>
-        <span style={{ marginLeft: `${progress}%` }}>
-          {progress}%
+        <span style={{ marginLeft: `${0}%` }}>
```

Comment:
how about writing `const progress = 0` until this feature has been fixed?

It'd tell me more about the purpose than the magic number of 0